python_modules/dagster/setup.py

```diff
@@ -69,7 +69,7 @@ def get_version() -> str:
             "alembic>=1.2.1,!=1.6.3,<1.7.0",
             "croniter>=0.3.34",
             "grpcio>=1.32.0",  # ensure version we require is >= that with which we generated the grpc code (set in dev-requirements)
-            "grpcio-health-checking>=1.32.0",
+            "grpcio-health-checking>=1.32.0,<1.44.0",
```

Comment:
if its just a bad push `!= 1.44.0` might be better but not clear it matters much for this package